app/views/works/_edit_javascript.html.erb

```diff
@@ -81,6 +81,31 @@
       return html;
     }
 
+    // Related Objects
+    // related_identifier:, related_identifier_type:, relation_type
+    var addRelatedObjectHtml = function(num, related_identifier, related_identifier_type, relation_type) {
```

Comment:
```suggestion
    var addRelatedObjectHtml = function({num, related_identifier, related_identifier_type, relation_type}) {
```
This is getting long enough that I'd worry about always getting the parameters in the correct order: Using parameter destructuring might be a little safer.

Reply:
or supply default values?
```suggestion
    var addRelatedObjectHtml = function({num, related_identifier='', related_identifier_type='', relation_type=''}) {
```

Reply:
(Might change the JS syntax in the future, but this is fine for now.)